selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/user/TestUserImpl.java

```diff
@@ -91,7 +91,7 @@ public TestUserImpl(
     this.userServiceClient.create(name, email, password);
     this.authToken = authServiceClient.login(name, password);
     this.id = userServiceClient.findByEmail(email).getId();
-    LOG.info("User name='{}', password='{}', id='{}' has been created", name, password, id);
+    LOG.info("User name='{}', id='{}' has been created", name, id);
```

Comment:
We need to have a password to be able to log into Che as a test user and verify the state of account.

Reply:
If password is configured why do you need to log it?

Reply:
Moreover, in context of testing of Che no test user has been created. Tests use pre-exist user which credentials are described in the test properties. So, the message is completely out of scope here.

But we need it for Codenvy Onprem and SaaS, which are using the same class **TestUserImpl**.